iib/web/static/api_v1.yaml

```diff
@@ -230,22 +230,24 @@ paths:
                 Done processing build request 1
         '404':
           description: Logs for build requests is not enabled in IIB
-            application/json:
```

Comment:
I think this line shouldn't change, but there should be a new line above it, `content:`, which has the same indentation as the line `description: ...`